test/cli.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const assert = require('assert');
+const execSync = require('child_process').execSync;
+
+describe('ecmarkup#cli', () => {
+  it('exits cleanly on success', () => {
+    execSync('./bin/ecmarkup.js test/example.html', { encoding: 'utf8' });
+  });
+
+  it('exits with an error on error', () => {
+    assert.throws(() => {
+      execSync('./bin/ecmarkup.js test/malformed.bad.html', { encoding: 'utf8' });
```

Comment:
lol does ecmarkup really write a ".bad.html" file on errors (－‸ლ)

Reply:
No, that's a file I added for the purposes of this test.

Reply:
ahhh ok cool :-)